boards/arm/nrf5340dk_nrf5340/CMakeLists.txt

```diff
@@ -70,6 +70,13 @@ if (CONFIG_BUILD_WITH_TFM)
 				$<TARGET_PROPERTY:tfm,TFM_S_BIN_FILE>
 				${CMAKE_BINARY_DIR}/tfm_s_signed.bin
 
+			# Generate an intel hex file from the signed output binary
```

Comment:
Can you append these commands to the `extra_post_build_commands` in a separate cmake statement?

To avoid merge conflicts when upmerging.

Reply:
I considered that, but since the commands depend on other post_build_commands, I thought we couldn't guarantee that they are run in the right order unless they are part of the same entry like this. Maybe @tejlmand can chime in on this ordering?

Reply:
Done